s3/main.tf

```diff
@@ -78,8 +79,13 @@ resource "aws_s3_bucket_lifecycle_configuration" "lifecycle-configuration" {
 resource "aws_kms_key" "kms-key" {
   count = var.config.serve_static_content ? 0 : 1
   # checkov:skip=CKV_AWS_7:We are not currently rotating the keys
-  description = "KMS Key for S3 encryption"
-  tags        = local.tags
+  description                        = "KMS Key for S3 encryption"
+  bypass_policy_lockout_safety_check = false
+  enable_key_rotation                = false
+  is_enabled                         = true
+  key_usage                          = "ENCRYPT_DECRYPT"
+  customer_master_key_spec           = "SYMMETRIC_DEFAULT"
```

Comment:
These are all just the default values anyway. Do we need them to be explicit?

Reply:
They were coming out as `null` in the tests.

Maybe that's a symptom of running the tests on a plan?

Reply:
Possibly.

Reply:
If that is the output from the plan, it's probably better to expect null rather than setting a bunch of variables when we don't need to or it looks like we've set them deliberately.

Reply:
I definitely don't want to explicitly check for `null` when that is not the actual expected result.

Should we even be testing for the default values on a plan if they are just coming back as null?

Reply:
Perhaps not. We should obviously test things we're explicitly setting. I think setting things to the default values to make tests pass is wrong, and it will lead to people wondering why we set them to those values in the future.